### ISSUE TITLE
Doesn't work in YouTube

### ISSUE THREAD

**Issue by flyingzebras** [tikservices/html5-video-everywhere#105]
HTML5 Video Everywhere! 0.4.8 doesn't work in YouTube for me, YouTube's default HTML5 player is use instead. Tested browsers:
* Firefox 58.0.2
* IceCat 52.3.0 -- The inbuilt version 0.2.38.1-signed didn't work either of course

**Comment by ghost** [comment]
This is true in Icecat 52.6.0 also

**Comment by jomo** [comment]
I, too, have noticed that – most of the time, but not always – the video player is not being replaced (using Firefox 60.0b1). The YouTube player is then left in a broken state, pressing the play button only plays a single frame and then pauses again.

**Comment by BenGodfrey78** [comment]
I have the same issue running WateFox 56. It just plays one frame. It sometimes works but sometimes when i go onto a  different page or reloads it it plays two video streams at one time. Also it will sometimes play the previous video I have now disabled it for YouTube. So I will be using the default YouTube player for now. Hope it gets fixed. Thanks

**Comment by BenGodfrey78** [comment]
UPDATE: For the last hour i've been playing arround. It seem to be working better now. I have just install uMatrix. I then block allot of stuff. The main one I think is the googlevideo dot com. I blocked everything but media and image. I think i could block images but when i did i had a little issue with anything showing up but that may be an issues with my setups. Also ad block is a must. I think YouTube video player forces, to some extent, the use of their player. Thanks for the good work dev.

**Comment by slrslr** [comment]
Yes, i also have problem on Youtube, video does not play. When i play with uBlock origin settings, i somehow make it play, but video shows one screen below. **TO FIX THIS ISSUE**, go to "about:addons" in FIrefox and in this extension settings, select to "Disable on Youtube".
